sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -814,24 +814,14 @@ private static BinaryData ReadAmqpDataBody(IEnumerable<Data> body)
         ///
         /// <returns>The set of bytes extracted from the <paramref name="data" />.</returns>
         ///
-        private static byte[] GetDataBytes(Data data)
+        private static ReadOnlyMemory<byte> GetDataBytes(Data data)
```

Comment:
```suggestion
private static ReadOnlyMemory<byte> GetDataBytes(Data data) =>
    data.Value switch
    {
        byte[] byteArray => byteArray,
        ArraySegment<byte> segment => segment,
        _ => ReadOnlyMemory<byte>.Empty
    };
```

The nittest of nits...  :smile: